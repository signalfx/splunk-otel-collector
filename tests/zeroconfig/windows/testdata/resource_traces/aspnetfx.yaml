resource_spans:
- attributes:
    deployment.environment: zc-iis-test
    host.name: <ANY>
    os.type: windows
    process.pid: <ANY>
    process.runtime.description: .NET Framework 4.8.4645.0
    process.runtime.name: .NET Framework
    process.runtime.version: "4.8"
    service.name: Default Web Site/aspnetfxapp
    splunk.distro.version: SPLUNK_OTEL_DOTNET_VERSION
    splunk.zc.method: splunk-otel-dotnet-vSPLUNK_OTEL_DOTNET_VERSION
    telemetry.distro.name: splunk-otel-dotnet
    telemetry.distro.version: SPLUNK_OTEL_DOTNET_VERSION
    telemetry.sdk.language: dotnet
    telemetry.sdk.name: opentelemetry
    telemetry.sdk.version: <ANY>
  scope_spans:
    - instrumentation_scope:
        attributes: {}
        name: OpenTelemetry.Instrumentation.AspNet.Telemetry
        version: <ANY>
      spans:
        - attributes:
            http.host: localhost
            http.method: GET
            http.route: api/{controller}/{id}
            http.status_code: 200
            http.target: /aspnetfxapp/api/values/4
            http.url: http://localhost/aspnetfxapp/api/values/4
            http.user_agent: Go-http-client/1.1
          name: api/{controller}/{id}
