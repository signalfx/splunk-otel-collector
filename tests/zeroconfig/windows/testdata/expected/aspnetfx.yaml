resourceSpans:
  - resource:
      attributes:
        - key: service.name
          value:
            stringValue: Default Web Site/aspnetfxapp
        - key: splunk.distro.version
          value:
            stringValue: 1.7.0
        - key: telemetry.distro.name
          value:
            stringValue: splunk-otel-dotnet
        - key: telemetry.distro.version
          value:
            stringValue: 1.7.0
        - key: os.type
          value:
            stringValue: windows
        - key: os.description
          value:
            stringValue: Microsoft Windows NT 10.0.20348.0
        - key: os.build_id
          value:
            stringValue: "20348"
        - key: os.name
          value:
            stringValue: Windows Server 2022 Datacenter
        - key: os.version
          value:
            stringValue: "6.3"
        - key: host.name
          value:
            stringValue: e137fad28e80.nxjcjlpip0jelm5h5wo3zvqkif.dx.internal.cloudapp.net
        - key: host.id
          value:
            stringValue: e5d95f12-d17e-4836-acdb-af990a55bf63
        - key: process.owner
          value:
            stringValue: DefaultAppPool
        - key: process.pid
          value:
            intValue: "2672"
        - key: process.runtime.description
          value:
            stringValue: .NET Framework 4.8.4749.0
        - key: process.runtime.name
          value:
            stringValue: .NET Framework
        - key: process.runtime.version
          value:
            stringValue: "4.8"
        - key: telemetry.sdk.name
          value:
            stringValue: opentelemetry
        - key: telemetry.sdk.language
          value:
            stringValue: dotnet
        - key: telemetry.sdk.version
          value:
            stringValue: 1.9.0
        - key: deployment.environment
          value:
            stringValue: zc-iis-test
        - key: splunk.zc.method
          value:
            stringValue: splunk-otel-dotnet-v1.8.0
    scopeSpans:
      - scope:
          name: OpenTelemetry.Instrumentation.AspNet.Telemetry
          version: 1.9.0-beta.1
        spans:
          - attributes:
              - key: server.address
                value:
                  stringValue: localhost
              - key: server.port
                value:
                  intValue: "80"
              - key: url.scheme
                value:
                  stringValue: http
              - key: http.request.method
                value:
                  stringValue: GET
              - key: network.protocol.version
                value:
                  stringValue: "1.1"
              - key: url.path
                value:
                  stringValue: /aspnetfxapp/api/values/4
              - key: user_agent.original
                value:
                  stringValue: Go-http-client/1.1
              - key: http.response.status_code
                value:
                  intValue: "200"
              - key: http.route
                value:
                  stringValue: api/{controller}/{id}
            endTimeUnixNano: "1728082061064807800"
            kind: 2
            name: GET api/{controller}/{id}
            parentSpanId: ""
            spanId: b09b6ad4bf1f170d
            startTimeUnixNano: "1728082060509344300"
            status: {}
            traceId: 23deab1db574f93ded7a4842e21a8c93